addons/project/tests/test_project_task_quick_create.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.addons.project.tests.test_project_base import TestProjectCommon
+from odoo.tests.common import Form
+
+class TestProjectTaskQuickCreate(TestProjectCommon):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        Users = cls.env['res.users'].with_context({'no_reset_password': True})
+        cls.user1 = Users.create({
+            'name': 'Raouf 1',
+            'login': 'raouf1',
+            'password': 'raouf1aa',
+            'email': 'raouf1@example.com',
+        })
+
+        cls.user2 = Users.create({
+            'name': 'Raouf 2',
+            'login': 'raouf2',
+            'password': 'raouf2aa',
+            'email': 'raouf2@example.com',
+        })
```

Comment:
```suggestion
        cls.user1, cls.user2 = cls.env['res.users'].with_context({'no_reset_password': True}).create([{
            'name': 'Raouf 1',
            'login': 'raouf1',
            'password': 'raouf1aa',
            'email': 'raouf1@example.com',
        }, {
            'name': 'Raouf 2',
            'login': 'raouf2',
            'password': 'raouf2aa',
            'email': 'raouf2@example.com',
        }])
```
We can batch that :)